src/features/game/lib/gameMachine.ts

```diff
@@ -1641,8 +1636,13 @@ export function startGame(authContext: AuthContext) {
                 1000 * 60 * 2,
               );
 
+              const leagueResultsInterval = setInterval(() => {
+                cb("CHECK_LEAGUE_RESULTS");
```

Comment:
Would it be possible here to follow the same pattern as the new season? It shows when the new day ticks over or when the player logs in after the season change. Would save having to run this every second.